samples/stock/demo/orderbook-chart/demo.js

```diff
@@ -0,0 +1,187 @@
+Highcharts.Templating.helpers.abs = value => Math.abs(value);
+
+function getRandomNumber(min, max) {
+    return Math.round(Math.random() * (max - min)) + min;
+}
+
+function generateBidAndAskData(n) {
+    const data = [[], []];
+    let bidPrice = getRandomNumber(29000, 30000),
+        askPrice = bidPrice + 0.5;
+    for (let i = 0; i < n; i++) {
+        bidPrice -= ((i * getRandomNumber(8, 10)));
+        askPrice += ((i * getRandomNumber(8, 10)));
+        data[0].push({
+            x: i,
+            y: (-i - 1) * getRandomNumber(70000, 110000),
+            price: bidPrice
+        });
+
+        data[1].push({
+            x: i,
+            y: (i + 1) * getRandomNumber(70000, 110000),
+            price: askPrice
+        });
+    }
+    return data;
+}
+
+const [bidsData, asksData] = generateBidAndAskData(10);
+
+function updateData(chart) {
+    const data = generateBidAndAskData(10);
+    chart.series.forEach((s, i) => {
+        s.setData(data[i], false);
+    });
+    chart.redraw();
+}
+
+Highcharts.chart('container', {
+    chart: {
+        animation: {
+            duration: 200
+        },
+        type: 'bar',
+        backgroundColor: '#23232f',
+        marginTop: 70,
+        events: {
+            load() {
+                setInterval(() => {
+                    if (this.series) {
+                        updateData(this);
+                    }
+                }, 200);
+            }
+        }
+    },
+
+    title: {
+        text: 'Order book live chart',
+        style: {
+            color: '#ffffff'
+        }
+    },
+
+    tooltip: {
+        headerFormat: 'Price: <b>{point.point.price}$</b></br>',
```

Comment:
Let's keep the same format for labels and tooltip. I mean including the same thousands separator:

![image](https://github.com/highcharts/highcharts/assets/1453926/e0cc7b5a-530b-4f46-8ad2-bd0a41fa0e0b)